test/Confluent.Kafka.IntegrationTests/Tests/Consumer_Drain.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+#pragma warning disable xUnit1026
+
+using System;
+using System.Threading;
+using Xunit;
+
+
+namespace Confluent.Kafka.IntegrationTests
+{
+    public partial class Tests
+    {
+        /// <summary>
+        ///     Test committing an offset when not subscribed to a group. This is valid provided there 
+        ///     are no active members of the group. If there is an active consumer group with the same
+        ///     groupId, the commit would be rejected (typically with an UNKNOWN_MEMBER_ID or
+        ///     ILLEGAL_GENERATION error).
```

Comment:
An additional negative test would be to try this when there is an existing conflicting group and verify that an error is raised and the offset is not committed.